boards/thingpulse_epulse_feather.json

```diff
@@ -0,0 +1,44 @@
+{
+  "build": {
+    "arduino":{
+      "ldscript": "esp32_out.ld"
+    },
+    "core": "esp32",
+    "extra_flags": [
+      "-DARDUINO_ESP32_DEV",
```

Comment:
Shouldn't it be `-DARDUINO_THINGPULSE_EPULSE_FEATHER`?

Reply:
Fixed it